app/application/use_cases/send_unlock_command.py

```
@@ -91,23 +151,28 @@ def execute(
         command_data["signature"] = signature
 
-        # 5. Send command via WebSocket
+        # Step 7: Send command via WebSocket connection
         message = {
             "type": MessageType.COMMAND.value,
             "payload": command_data,
         }
 
         try:
+            # Fire-and-forget: send without waiting for acknowledgment
             asyncio.create_task(manager.send_to_vault(vault_id, message))
```

Ruff (0.14.14)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)